parsers/rhea/rhea.go

```diff
@@ -0,0 +1,742 @@
+				switch containsx.XMLName.Local {
+				case "containsN":
+					newReactionParticipant = ReactionParticipant{
+						ReactionSide: description.About,
+						Contains:     1,
+						ContainsN:    true,
+						Minus:        false,
+						Plus:         false,
+						Accession:    containsx.Content}
+				case "contains2n":
+					newReactionParticipant = ReactionParticipant{
+						ReactionSide: description.About,
+						Contains:     2,
+						ContainsN:    true,
+						Minus:        false,
+						Plus:         false,
+						Accession:    containsx.Content}
+				case "containsNminus1":
+					newReactionParticipant = ReactionParticipant{
+						ReactionSide: description.About,
+						Contains:     1,
+						ContainsN:    true,
+						Minus:        true,
+						Plus:         false,
+						Accession:    containsx.Content}
+				case "containsNplus1":
+					newReactionParticipant = ReactionParticipant{
+						ReactionSide: description.About,
+						Contains:     1,
+						ContainsN:    true,
+						Minus:        false,
+						Plus:         true,
+						Accession:    containsx.Content}
+				default:
+					i, err := strconv.Atoi(containsx.XMLName.Local[8:])
+					if err != nil {
+						return Rhea{}, err
+					}
+					newReactionParticipant = ReactionParticipant{
+						ReactionSide: description.About,
+						Contains:     i,
+						ContainsN:    false,
+						Minus:        false,
+						Plus:         false,
+						Accession:    containsx.Content}
+				}
+				newReactionParticipant.Compound = compoundParticipantMap[description.About]
+				rhea.ReactionParticipants = append(rhea.ReactionParticipants, newReactionParticipant)
+			}
+		}
+	}
+	return rhea, nil
+}
+
+// ReadRhea reads in a a gzip'd Rhea dump (https://www.rhea-db.org/help/download) into bytes.
+func ReadRhea(gzipPath string) ([]byte, error) {
+	// Get gz'd file bytes
+	xmlFile, err := os.Open("data/rhea.rdf.gz")
+	if err != nil {
+		return []byte{}, err
+	}
+
+	// Decompress gz'd file
+	r, err := gzip.NewReader(xmlFile)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	// Read decompressed gz'd file
+	rheaBytes, err := ioutil.ReadAll(r)
+	if err != nil {
+		return []byte{}, err
+	}
+	return rheaBytes, nil
+}
+
+/******************************************************************************
+
+Database functions
+
+These functions take in the rhea.rdf.gz dump file and import them into an sqlite
+database. This database can be used locally for mapping functions, but more importantly,
+it is used because it can enforce the relationships between each part of Rhea and alert
+the user if the parser is failing to pick up on any of the relationships in the Rhea
+database.
+
+******************************************************************************/
+
+// CreateRheaSqlite initializes a SQLite database with the proper tables to have Rhea inserted using InsertRheaSqlite.
+func CreateRheaSqlite(db *sql.DB) error {
+	schema := `
+	-- Rhea tables --
+	CREATE TABLE chebi (
```

Comment:
```suggestion
	CREATE TABLE IF NOT EXISTS chebi (
```
 to make this idempotent.
 
 Similar for the rest of the `CREATE TABLE` statements.